moziris/api/settings.py

```diff
@@ -74,7 +75,7 @@ class _Settings:
     DEFAULT_UI_DELAY_SHORT = 0.5
     DEFAULT_UI_DELAY_LONG = 2.5
     DEFAULT_SYSTEM_DELAY = 5
-    PACKAGE_ROOT = os.path.realpath(os.path.split(__file__)[0] + "/../..")
+    PACKAGE_ROOT = os.getenv("IRIS_CODE_ROOT")
```

Comment:
This isn't right. Package root is not necessarily equal to code root. Package root refers to the relative path from this file, guaranteeing a path to the package. Code root will depend on how this library is deployed, and it's likely different. I'd revert this line back to the original.